samigo/samigo-app/src/webapp/jsf/evaluation/totalScores.jsp

```diff
@@ -276,7 +276,8 @@ $(document).ready(function(){
        <h:outputText value="\"></a>" escape="false" />
 
        <h:commandLink styleClass="sam-scoretable-deleteattempt" title="#{commonMessages.delete_attempt}" action="totalScores" immediate="true" rendered="true" >
-         <h:outputText value="X" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" />
+           <h:panelGroup rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}"><span class="fa fa-trash" aria-hidden="true"></span></h:panelGroup>
+	   <span class="sr-only"><h:outputText value="#{commonMessages.delete}" rendered="#{description.submittedDate!=null &&  description.assessmentGradingId ne '-1'}" /></span>
```

Comment:
would be simpler to just move the span and outputText inside the panelGroup on line above... then you just have one if statement